### ISSUE TITLE
Project counter missing, again

### ISSUE THREAD

**Issue by KazzmanK** [go-gitea/gitea#28052]
### Description

After #26826, project counter at organization level is still missing

At repository level all counters are in place

### Screenshots

![image](https://github.com/go-gitea/gitea/assets/13328513/6fe942e8-327f-4f19-855c-bcbf35553d08)

### Gitea Version

1.21

### Can you reproduce the bug on the Gitea demo site?

No

### Browser Version

119

**Comment by yp05327** [comment]
#26826 fixed another issue, didn't fix this. 
It will be fixed in #28068.
